### DISCUSSION TITLE
Créer un cluster kubernetes avec vos propres serveurs

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#221] [Announcements]
# Créer un cluster kubernetes avec vos propres serveurs

Tout le monde n'a pas les moyens de s'offrir un cluster sur le Cloud! Nous allons donc apprendre à créer un cluster kubernetes avec nos propres serveurs.

https://zerofiltre.tech/articles/167-creer-un-cluster-kubernetes-avec-vos-propres-serveurs

**Comment by mcgivrer** [comment]
Bonjour, votre article semble très intéressant, cependant sa mise en forme est défectueuse, rendant la lecture des fichiers yaml illisible.
Pourriez-vous procédé a la correction ?  par avance merci !
